source/includes/localhost-connection.rst

```diff
@@ -1,16 +1,13 @@
-.. note:: Connect to localhost
+To connect to a database running locally on its default port, change the
+connection string to ``"mongodb://localhost"``. 
 
-    If you want to connect to a database server running locally on its
-    default port, change the Atlas connection string to
-    ``"mongodb://localhost"``.
+To connect to a specific port, change the connection string to
+``"mongodb://localhost:<port_number>"``. 
```

Comment:
I think the example weakens these two statements because `27017` is the default port but you are using the connection string from the second item.

Maybe just:
`To connect to a database running locally, change the connection string to mongodb://localhost:<port>`

And then let the example stand.